packages/repository/package-lock.json

```diff
@@ -4,23 +4,499 @@
+			"integrity": "sha512-SnA9aLbyOCcnnbQEGwdfBggnc142h/rbqqsXcaATj2hZcegCl903pUD/lfpsNBlBSuWow/YDfRyJuWi2EPR5cg==",
+			"dev": true,
+			"requires": {
+				"@babel/code-frame": "^7.0.0",
+				"@babel/generator": "^7.5.0",
+				"@babel/helper-function-name": "^7.1.0",
+				"@babel/helper-split-export-declaration": "^7.4.4",
+				"@babel/parser": "^7.5.0",
+				"@babel/types": "^7.5.0",
+				"debug": "^4.1.0",
+				"globals": "^11.1.0",
+				"lodash": "^4.17.11"
+			}
+		},
+		"@babel/types": {
+			"version": "7.5.0",
+			"resolved": "https://registry.npmjs.org/@babel/types/-/types-7.5.0.tgz",
+			"integrity": "sha512-UFpDVqRABKsW01bvw7/wSUe56uy6RXM5+VJibVVAybDGxEW25jdwiFJEf7ASvSaC7sN7rbE/l3cLp2izav+CtQ==",
+			"dev": true,
+			"requires": {
+				"esutils": "^2.0.2",
+				"lodash": "^4.17.11",
+				"to-fast-properties": "^2.0.0"
+			}
+		},
+		"@hapi/address": {
+			"version": "2.0.0",
+			"resolved": "https://registry.npmjs.org/@hapi/address/-/address-2.0.0.tgz",
+			"integrity": "sha512-mV6T0IYqb0xL1UALPFplXYQmR0twnXG0M6jUswpquqT2sD12BOiCiLy3EvMp/Fy7s3DZElC4/aPjEjo2jeZpvw==",
+			"dev": true
+		},
+		"@hapi/hoek": {
+			"version": "6.2.4",
+			"resolved": "https://registry.npmjs.org/@hapi/hoek/-/hoek-6.2.4.tgz",
+			"integrity": "sha512-HOJ20Kc93DkDVvjwHyHawPwPkX44sIrbXazAUDiUXaY2R9JwQGo2PhFfnQtdrsIe4igjG2fPgMra7NYw7qhy0A==",
+			"dev": true
+		},
+		"@hapi/joi": {
+			"version": "15.1.0",
+			"resolved": "https://registry.npmjs.org/@hapi/joi/-/joi-15.1.0.tgz",
+			"integrity": "sha512-n6kaRQO8S+kepUTbXL9O/UOL788Odqs38/VOfoCrATDtTvyfiO3fgjlSRaNkHabpTLgM7qru9ifqXlXbXk8SeQ==",
+			"dev": true,
+			"requires": {
+				"@hapi/address": "2.x.x",
+				"@hapi/hoek": "6.x.x",
+				"@hapi/marker": "1.x.x",
+				"@hapi/topo": "3.x.x"
+			}
+		},
+		"@hapi/marker": {
+			"version": "1.0.0",
+			"resolved": "https://registry.npmjs.org/@hapi/marker/-/marker-1.0.0.tgz",
+			"integrity": "sha512-JOfdekTXnJexfE8PyhZFyHvHjt81rBFSAbTIRAhF2vv/2Y1JzoKsGqxH/GpZJoF7aEfYok8JVcAHmSz1gkBieA==",
+			"dev": true
+		},
+		"@hapi/shot": {
+			"version": "4.1.0",
+			"resolved": "https://registry.npmjs.org/@hapi/shot/-/shot-4.1.0.tgz",
+			"integrity": "sha512-rpUU5cF08fqAZLLnue6Sy0osj1QMPbrYskehxtLFPdk7CwlPcu9N/wRtgu7vDHTQCKTkag6M8sjc8V8p8lSxpg==",
+			"dev": true,
+			"requires": {
+				"@hapi/hoek": "6.x.x",
+				"@hapi/joi": "15.x.x"
+			}
+		},
+		"@hapi/topo": {
+			"version": "3.1.2",
+			"resolved": "https://registry.npmjs.org/@hapi/topo/-/topo-3.1.2.tgz",
+			"integrity": "sha512-r+aumOqJ5QbD6aLPJWqVjMAPsx5pZKz+F5yPqXZ/WWG9JTtHbQqlzrJoknJ0iJxLj9vlXtmpSdjlkszseeG8OA==",
+			"dev": true,
+			"requires": {
+				"@hapi/hoek": "8.x.x"
+			},
+			"dependencies": {
+				"@hapi/hoek": {
+					"version": "8.0.2",
+					"resolved": "https://registry.npmjs.org/@hapi/hoek/-/hoek-8.0.2.tgz",
+					"integrity": "sha512-O6o6mrV4P65vVccxymuruucb+GhP2zl9NLCG8OdoFRS8BEGw3vwpPp20wpAtpbQQxz1CEUtmxJGgWhjq1XA3qw==",
+					"dev": true
+				}
+			}
+		},
+		"@loopback/build": {
```

Comment:
`@loopback/build` is a dependency that should be resolved within the monorepo. Please fix.